components/combobox/index.css

```diff
@@ -0,0 +1,609 @@
+
+    &::placeholder {
+      color: var(--highcontrast-combobox-font-color-placeholder, var(--mod-combobox-font-color-hover, var(--spectrum-combobox-font-color-hover)));
+    }
+  }
+
+  &:focus,
+  &.is-focused {
+    background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-focus, var(--spectrum-combobox-background-color-focus)));
+    border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-focus, var(--spectrum-combobox-border-color-focus)));
+    color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-focus, var(--spectrum-combobox-font-color-focus)));
+
+    &::placeholder {
+      color: var(--highcontrast-combobox-font-color-placeholder, var(--mod-combobox-font-color-focus, var(--spectrum-combobox-font-color-focus)));
+    }
+  }
+
+  &:focus:hover,
+  &.is-focused:hover {
+    background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-focus-hover, var(--spectrum-combobox-background-color-focus-hover)));
+    border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-focus-hover, var(--spectrum-combobox-border-color-focus-hover)));
+    color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-focus-hover, var(--spectrum-combobox-font-color-focus-hover)));
+  }
+}
+
+.spectrum-Combobox--quiet {
+  border-radius: var(--mod-combobox-quiet-fieldbutton-border-radius, var(--spectrum-combobox-quiet-fieldbutton-border-radius));
+
+  .spectrum-Combobox-textfield {
+    &.is-keyboardFocused::after {
+      pointer-events: none;
+      content: '';
+      position: absolute;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      top: 0;
+      margin: calc(
+        (var(--mod-combobox-focus-indicator-gap, var(--spectrum-combobox-focus-indicator-gap)) +
+         var(--mod-combobox-border-width, var(--spectrum-combobox-border-width))
+         * -1)
+      );
+      inline-size: 100%;
+      block-size: calc(
+        100% +
+        (var(--mod-combobox-focus-indicator-gap, var(--spectrum-combobox-focus-indicator-gap)) * 2) +
+        (var(--mod-combobox-border-width, var(--spectrum-combobox-border-width)) * 2)
+      );
+      margin: auto;
+      border: 0;
+      border-bottom-style: solid;
+      border-bottom-width: var(--mod-combobox-focus-indicator-thickness, var(--spectrum-combobox-focus-indicator-thickness));
+      border-bottom-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-focus-indicator-color, var(--spectrum-combobox-focus-indicator-color)));
+      border-radius: 0;
+    }
+  }
+
+  .spectrum-Combobox-button {
+    inline-size: auto;
+
+    .spectrum-PickerButton-fill {
+      border: none;
+      background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-default, var(--spectrum-combobox-background-color-default)));
```

Comment:
This background-color is not being used. It's being overwritten by a value from PickerButton. I don't know if that's a bad thing or not but just a heads up